web/src/main/kotlin/no/nav/su/se/bakover/web/routes/sak/SakJson.kt

```diff
@@ -55,16 +56,37 @@ internal data class SakJson(
                 )
             },
             utbetalingerKanStansesEllerGjenopptas = utbetalingstidslinje().tidslinje.let {
-                when {
-                    it.isEmpty() -> KanStansesEllerGjenopptas.INGEN
-                    it.last() is UtbetalingslinjePåTidslinje.Stans -> KanStansesEllerGjenopptas.GJENOPPTA
-                    else -> KanStansesEllerGjenopptas.STANS
+                if (it.isNotEmpty() && ingenOpphørEllerHull(it)) {
+                    if (it.last() is UtbetalingslinjePåTidslinje.Stans) {
+                        KanStansesEllerGjenopptas.GJENOPPTA
+                    } else {
+                        KanStansesEllerGjenopptas.STANS
+                    }
+                } else {
+                    KanStansesEllerGjenopptas.INGEN
                 }
             },
             revurderinger = revurderinger.map { it.toJson() },
             vedtak = vedtakListe.map { it.toJson() },
-            klager = klager.map { it.toJson() }
+            klager = klager.map { it.toJson() },
         )
+
+        /**
+         * Tillater ikke stans dersom stønadsperiodene inneholder opphør eller hull, siden det kan bli problematisk
+         * hvis man stanser og gjenopptar på tvers av nevnte opphør og hull. Ta opp igjen problemstillingen dersom
+         * fag trenger å stanse på et slikt tilfelle.
+         */
+        private fun ingenOpphørEllerHull(utbetalingslinjer: List<UtbetalingslinjePåTidslinje>): Boolean {
+            if (utbetalingslinjer.any { it is UtbetalingslinjePåTidslinje.Opphør }) {
+                return false
+            }
+
+            if (utbetalingslinjer.map { linje -> linje.periode }.reduser().size > 1) {
```

Comment:
Dette vil vel føre til at vi aldri kommer til å kunne stans utbetalinger i fremtiden, dersom det f.eks eksisterer opphold mellom to stønadsperioder 2 år tilbake i tid?